.gitignore

```diff
@@ -1 +1,2 @@
 /Debug/*
```

Comment:
Change this back to include the *. If the asterisk is not there, we'll run into a bunch of problems when pushing/pulling